src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAroundCheck.java

```diff
@@ -473,6 +482,16 @@ public void setAllowEmptyCatches(boolean allow) {
         allowEmptyCatches = allow;
     }
 
+    /**
+     * Setter to allow empty switch blocks and block statements.
+     *
+     * @param allow {@code true} to allow empty switch case and default blocks.
+     * @since 10.18.3
```

Comment:
New property can not be bumped in bug fix version.
We need 10.19

Reply:
done